plugin/trino-hive/src/test/java/io/trino/plugin/hive/benchmark/StandardFileFormats.java

```diff
@@ -0,0 +1,350 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.hive.benchmark;
+
+import com.google.common.collect.ImmutableMap;
+import io.trino.orc.OrcReaderOptions;
+import io.trino.orc.OrcWriter;
+import io.trino.orc.OrcWriterOptions;
+import io.trino.orc.OrcWriterStats;
+import io.trino.orc.OutputStreamOrcDataSink;
+import io.trino.orc.metadata.OrcType;
+import io.trino.parquet.writer.ParquetSchemaConverter;
+import io.trino.parquet.writer.ParquetWriter;
+import io.trino.parquet.writer.ParquetWriterOptions;
+import io.trino.plugin.hive.FileFormatDataSourceStats;
+import io.trino.plugin.hive.HdfsEnvironment;
+import io.trino.plugin.hive.HiveCompressionCodec;
+import io.trino.plugin.hive.HiveConfig;
+import io.trino.plugin.hive.HivePageSourceFactory;
+import io.trino.plugin.hive.HiveRecordCursorProvider;
+import io.trino.plugin.hive.HiveStorageFormat;
+import io.trino.plugin.hive.orc.OrcPageSourceFactory;
+import io.trino.plugin.hive.parquet.ParquetPageSourceFactory;
+import io.trino.plugin.hive.parquet.ParquetReaderConfig;
+import io.trino.plugin.hive.rcfile.RcFilePageSourceFactory;
+import io.trino.rcfile.binary.BinaryRcFileEncoding;
+import io.trino.rcfile.text.TextRcFileEncoding;
+import io.trino.spi.Page;
+import io.trino.spi.connector.ConnectorSession;
+import io.trino.spi.type.Type;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.List;
+import java.util.Optional;
+
+import static io.trino.orc.OrcWriteValidation.OrcWriteValidationMode.BOTH;
+import static io.trino.plugin.hive.HiveTestUtils.TYPE_MANAGER;
+import static io.trino.plugin.hive.HiveTestUtils.createGenericHiveRecordCursorProvider;
+import static org.joda.time.DateTimeZone.UTC;
+
+public final class StandardFileFormats
```

Comment:
Make this class implement abstract `FileFormat` and implement default methods instead of having them as defaults in `FileFormat`